demos/build_demos.sh

```diff
@@ -26,12 +26,18 @@ error() {
 trap 'error ${LINENO}' ERR
 
 extra_cmake_opts=()
+build_targets=()
 
 for opt in "$@"; do
     case "$opt" in
     -DENABLE_PYTHON=*)
         extra_cmake_opts+=("$opt")
         ;;
+    --target=*)
+        tmp="${opt%\"}"
+        tmp="${tmp#\"}"
```

Comment:
What is this for?

Reply:
It removes quotation marks. In `*.bat` file for it we use `%~2`, I didn't find better solution here to do the same.

Reply:
But why would there be any quotation marks in the first place? If the user runs something like  `build_demos.sh --target="a b"`, then those quotes are shell syntax and will not be present in the value of `opt`.